src/components/views/rooms/RoomNameEditor.js

```diff
@@ -34,7 +34,7 @@ export default createReactClass({
         };
     },
 
-    componentWillMount: function() {
+    componentDidMount: function() {
```

Comment:
We assume `this._placeholderName` is set by first render, this would mean it is not.